WordPress/Classes/ViewRelated/Reader/ReaderTagStreamHeader.swift

```diff
@@ -31,8 +25,13 @@ import WordPressShared
 
     @objc open func configureHeader(_ topic: ReaderAbstractTopic) {
         titleLabel.text = topic.title
-        WPStyleGuide.applyReaderFollowButtonStyle(followButton)
         followButton.isSelected = topic.following
+
+        WPStyleGuide.applyReaderFollowTopicButtonStyle(followButton)
+        if #available(iOS 13, *) {
+            traitCollection.performAsCurrent { followButton.layer.borderColor = UIColor.gray(.shade30).cgColor }
```

Comment:
Can we extract the color here to a constant?

Also, you might want to do the same logic for the `tintedFollowingIcon`. When toggline dark/light mode the icon remains with the same color:

| Light  | Dark |
| ------------- | ------------- |
| <img width="556" alt="Screen Shot 2020-07-27 at 11 06 10" src="https://user-images.githubusercontent.com/7040243/88551620-6fef9500-cff9-11ea-9012-604e3fe9d3da.png"> | <img width="556" alt="Screen Shot 2020-07-27 at 11 07 37" src="https://user-images.githubusercontent.com/7040243/88551636-767e0c80-cff9-11ea-9329-1f19c4b43713.png"> |

Reply:
Following DM discussion: I think it's weird to do a special handling for the following icon (the follow icon works as expected). For the sake of testing though I did:
``` 
traitCollection.performAsCurrent {
                let side = WPStyleGuide.fontSizeForTextStyle(.headline)
                let size = CGSize(width: side, height: side)
                let followingIcon = UIImage.gridicon(.readerFollowing, size: size)
             followButton.setImage(followingIcon.imageWithTintColor(WPStyleGuide.FollowButton.Style.followingTextColor), for: .selected)
                setNeedsDisplay()
}
``` 
but it didn't work :(

Reply:
@yaelirub I managed to make this work. :) 

Here's the patch: https://gist.github.com/leandroalonso/3ad0e6044f4cd458646851dde73a5476